Makefile

```diff
@@ -13,11 +16,13 @@ clean:
 	rm -rf $(BUILD_DIR)
 
 .PHONY: b2d_iso
-b2d_iso: ISO_NAME=minishift-b2d
+b2d_iso: OSRELEASE_FILE         = os-release
```

Comment:
Why don't you just set these variables? There is no need in this Makefile to do this via a target. It just gets complicated. The reason we had this for CentOS is that we generate two different ISO from the same repo.